pseudomonas_analysis/viz_E-GEOD-33245_cbrB_v_WT.py

```diff
@@ -242,8 +242,19 @@ def make_volcano_template_highlight_genelist(
 cbrB_crc_df.loc[cbrB_crc_df["Gene Name"] == "cbrB", "gene group"] = "cbrB"
 cbrB_crc_df.loc[cbrB_crc_df["Gene Name"] == "crc", "gene group"] = "crc"
 
+print(cbrB_crc_df.shape)
 cbrB_crc_df.head()
 
+# +
+# Drop cbrB and crc rows for plot
```

Comment:
Maybe you could comment why you're doing this here? Something like what you said in the PR description would be fine, just so you remember in the future why you dropped these rows.

Reply:
Very good idea. I'll add this